presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -200,7 +200,7 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
                 connection.getCatalog(),
                 escapeNamePattern(schemaName, escape).orElse(null),
                 escapeNamePattern(tableName, escape).orElse(null),
-                new String[] {"TABLE", "VIEW", "MATERIALIZED VIEW", "FOREIGN TABLE"});
+                new String[] {"TABLE", "VIEW", "MATERIALIZED VIEW", "FOREIGN TABLE", "SYSTEM TABLE", "SYSTEM VIEW"});
```

Comment:
pg_catalog.xxx are SYSTEM TABLEs, VIEWs or mix?

Reply:
pg_catalog.xxx are mix of them and it's not easy to tell simply by their names also. ex, `pg_foreign_table` is SYSTEM TABLE but `pg_tables` is SYSTEM VIEW